addons/medical_damage/stringtable.xml

```diff
@@ -697,5 +697,37 @@
             <Polish>Szansa że osoba straci przytomność gdy jej ból jest powyżej tolerowalnego progu podczas otrzymywania obrażeń.</Polish>
             <Japanese>ユニットがダメージを受けた時に痛みが許容しきい値を超えた場合、気絶をする確立を設定します。</Japanese>
         </Key>
+        <Key ID="STR_ACE_Medical_Damage_enableVehicleCrashes_DisplayName">
+            <English>Vehicle Crashes</English>
+            <Russian>Аварии транспорта</Russian>
+            <Polish>Obrażenia od kolizji</Polish>
+            <Spanish>Accidentes de vehículos</Spanish>
+            <German>Fahrzeugunfälle</German>
+            <Czech>Poškození z kolize</Czech>
+            <Portuguese>Colisão de veículos</Portuguese>
+            <French>Accident en véhicule</French>
+            <Hungarian>Járműbalesetek</Hungarian>
+            <Italian>Schianto Veicoli</Italian>
+            <Japanese>車両の衝突</Japanese>
+            <Korean>차량 사고</Korean>
+            <Chinesesimp>载具碰撞</Chinesesimp>
+            <Chinese>載具碰撞</Chinese>
+        </Key>
+        <Key ID="STR_ACE_Medical_Damage_enableVehicleCrashes_Description">
+            <English>Do units take damage from a vehicle crash?</English>
```

Comment:
Setting descriptions should not be questions. We tried to get rid of those with the medical rewrite.

Reply:
Will adjust, but can only translate for English at that point.